### PR TITLE
events verification

### PR DESCRIPTION
**Pull request by m4ksio**
Adds an example code how to verify correctness and completeness of events

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -19,9 +18,10 @@ This package contains code samples that interact with the [Flow Emulator](https:
     - [Single Party, Single Signature](#single-party-single-signature)
     - [Single Party, Multiple Signatures](#single-party-multiple-signatures)
     - [Multiple Parties](#multiple-parties)
-    - [Multiple Parties, Two autorizers](#multiple-parties-two-authorizers)
+    - [Multiple Parties, Two autorizers](#multiple-parties-two-autorizers)
```
Comment (Kay-Zee) [inline examples/README.md:21]: is this right? i thought this should be `authorizers`

**Review by Kay-Zee — APPROVED** [review]

File: examples/README.md (hunk 1 above)
Comment (m4ksio) [reply examples/README.md:21]: Hmm, this part if autogenerated, I guess there is typo if the document, let me fix it

File: execution_result.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+type ExecutionResult struct {
+	PreviousResultID Identifier // commit of the previous ER
+	BlockID          Identifier // commit of the current block
+	Chunks           []*Chunk
+	ServiceEvents    []*ServiceEvent
+}
+
+type Chunk struct {
+	CollectionIndex      uint
```
Comment (sideninja) [inline execution_result.go:13]: Shouldn't this be defined as uint32?
```suggestion
	CollectionIndex      uint32
```

File: client/client.go [hunk 3]
```diff
@@ -556,3 +552,42 @@ func (c *Client) GetLatestProtocolStateSnapshot(ctx context.Context, opts ...grp
...
+	if err != nil {
+		return nil, newRPCError(err)
+	}
+
+	chunks := make([]*flow.Chunk, len(er.ExecutionResult.Chunks))
+	serviceEvents := make([]*flow.ServiceEvent, len(er.ExecutionResult.ServiceEvents))
+
+	for i, chunk := range er.ExecutionResult.Chunks {
+		chunks[i] = &flow.Chunk{
+			CollectionIndex:      uint(chunk.CollectionIndex),
```
Comment (sideninja) [inline client/client.go:569]:
```suggestion
			CollectionIndex:      uint32(chunk.CollectionIndex),
```

**Review by janezpodhostnik — APPROVED** [review]

File: execution_result.go (hunk 2 above)
Comment (m4ksio) [reply execution_result.go:13]: In `flow-go` it is `uint`

File: client/client.go (hunk 3 above)
Comment (m4ksio) [reply client/client.go:569]: same

**Review by sideninja — APPROVED** [review]
👍

**Pull request merged**